classes/class_os.rst

```diff
@@ -17,7 +17,7 @@ Operating System functions.
 Description
 -----------
 
-Operating System functions. OS wraps the most common functionality to communicate with the host operating system, such as the clipboard, video driver, date and time, timers, environment variables, execution of binaries, command line, etc.
```

Comment:
I wasn't sure if it was also appropriate remove the `timers, ` There are still two `dealy_` functions in OS, so I think it's partly appropriate to stay